addons/event/static/src/scss/event_badge_report.scss

```diff
@@ -24,34 +24,53 @@
         color: #939393;
```

Comment:
Remove "event_foldable_badge_report.css" that is duplicated inside this commit (with small explanation).